src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/Tov.cpp

```diff
@@ -220,17 +216,15 @@ void TovSolution::integrate(
       // Transform observed radius to isotropic, so we use the isotropic radius
       // for all interpolations below
       observer.radius[i] /= square(observer.conformal_factor[i]);
-      // The interpolation is not safe otherwise
     }
+    // The interpolation is not safe otherwise
```

Comment:
(comment on arbitrary line) @isaaclegred Why did you have to increase the tolerance in `test_tov_dp_dr`?: https://github.com/sxs-collaboration/spectre/pull/4111/files#diff-ef62474f8d2a8b459659b395a68f36b1e6b0c8692373bbe2f7b2a4ad599c818bR165

Reply:
I don't remember this. Look at the link I wouldn't expect it to need to be changed. Maybe something that ended up left over from debugging?

Reply:
I think it is because the accuracy we got probably dropped in this case when we switched to cubic splines.  In practice this wasn't a big deal because the solution-to-grid discretization error is going to always be worse, but it may be a good motivation to try to find a way to bring back barycentric rational interpolation, if this turns out to be generically true.